pumpkin-world/src/item/item_category_checks.rs

```diff
@@ -0,0 +1,75 @@
+use crate::item::Item;
```

Comment:
I don't like name of this file. I think item_category would be better

Reply:
maybe `item_categories`?